src/cli/driver.ts

```diff
@@ -53,6 +55,12 @@ export function runServer() {
   new PlaywrightDispatcher(dispatcherConnection.rootDispatcher(), playwright);
 }
 
+export async function launchBrowserServer(browserName: string) {
+  const browserType = (playwright as any)[browserName] as BrowserType;
+  const server = await browserType.launchServer();
```

Comment:
This looks good, but i would fs.readFileSync(config) and pass it here. That way user can opt into accepting downloads, respecting proxy, etc.

Reply:
Done.